### PR TITLE
Update the color theme

### PR DESCRIPTION
**Pull request by hkiyomaru**
## What

Update the color theme by overriding the bootstrap variables.

## Screenshots

image

### CODE REVIEW

**Comment by hkiyomaru** [comment]
image

**Comment by hkiyomaru** [comment]
image

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import 'bootstrap/dist/css/bootstrap.min.css'
 import '@src/styles/global.css'
+import '@src/styles/bootstrap.scss'
```
Comment (honai) [inline src/pages/_app.tsx:2]: Styles should be applied in the order of Bootstrap-> Custom Bootstrap-> our own global.css.  Bootstrap shouldn't overwrite styles in global.css, so I want you to reverse the import order.

**Review by honai — CHANGES_REQUESTED** [review]
Thanks!  Please fix the import order.

File: src/pages/_app.tsx (hunk 1 above)
Comment (hkiyomaru) [reply src/pages/_app.tsx:1]: Fixed.

**Review by honai — APPROVED** [review]

**Pull request merged**